packages/edit-widgets/src/components/customizer-edit-widgets-initializer/index.js

```diff
@@ -2,24 +2,30 @@
  * WordPress dependencies
  */
 import { navigateRegions } from '@wordpress/components';
+import { useSimulatedMediaQuery } from '@wordpress/block-editor';
 import { __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import './sync-customizer';
+import Header from '../header';
 import WidgetAreasBlockEditorProvider from '../widget-areas-block-editor-provider';
 import WidgetAreasBlockEditorContent from '../widget-areas-block-editor-content';
 
 function CustomizerEditWidgetsInitializer( { settings } ) {
+	useSimulatedMediaQuery( 'resizable-editor-section', 360 );
 	return (
-		<WidgetAreasBlockEditorProvider blockEditorSettings={ settings }>
+		<WidgetAreasBlockEditorProvider
+			blockEditorSettings={ { ...settings, hasFixedToolbar: true } }
```

Comment:
We should probably memoize this prop since it's such a big tree of components.

Reply:
I added memoization as suggested 👍